bittide-instances/src/Bittide/Instances/ClockControl.hs

```diff
@@ -22,4 +22,7 @@ callisto3 ::
   -- | Speed change requested from clock multiplier
   Signal Basic200 SpeedChange
 callisto3 clk rst ena dataCounts =
-  callistoClockControl clk rst ena config dataCounts
+  callistoClockControl clk rst ena config availableLinkMask dataCounts
+ where
+  -- all links available
+  availableLinkMask = pure $ v2bv $ map (const high) indicesI
```

Comment:
```suggestion
  availableLinkMask = pure maxBound
```

Reply:
I like and dislike your suggestion at the same time. I like that its short and looks smart. What i don't like is that it assumes unsigned integer semantics for the vector. You could also assume signed integer semantics for the vector and suddenly your solution turns wrong.

In general, I guess there shouldn't be something like `min` and `max` for pure bitvectors. Do you really want to introduce the change?

Reply:
Yeah, I can see your point. The documentation from `BitVector` states that `Num` operations (and I suppose all others, though that's not really documented) should be interpreted as `Unsigned`. It's a pretty wide-spread pattern within Clash, so I personally would be perfectly fine with it.

Otherwise: `pack (repeat high)` should also work.

Reply:
To your pick :)

Reply:
That one's much better.